### ISSUE TITLE
Code for producing paper evaluation results

### ISSUE THREAD

**Issue by griffbr** [zhangxin-xd/Dataset-Pruning-TDDS#1]
Hi Zhang,

First, fantastic work and paper (and great speaking with you at CVPR)! Thank you for making this code available to reproduce results.

Second, do you plan to release evaluation code to generate the final experiment results in your paper? From the three current scripts (`train`, `importance_evaluation`, and `train_subset`), I only get the validation plots from `train_subset.py` (see example from quick training run below).

![curve](https://github.com/zhangxin-xd/Dataset-Pruning-TDDS/assets/26256043/c206157c-8247-4119-b1fc-19ea192e4f9a)

**Comment by zhangxin-xd** [comment]
Hi, thank you for your interest! 😊 The evaluation accuracy reported in the paper corresponds to the validation accuracy achieved after training on the subset. You can reproduce this by using the subset training command.
